Wrappers.h

```diff
@@ -0,0 +1,78 @@
+//
+// Created by james on 12/05/2021.
+//
+
+#include "AmplitudeEnvelope.h"
+#include "HighRise.h"
+#include "Chord.h"
+#include "Button.h"
+#include "AmpModule.h"
```

Comment:
Don't include what you don't use.
You aren't using any thing from these files IN THESE HEADER FILES.
That means you can forward declare all of the types
Include the header files in the relevant cpp file
- [ ] revise